src/components/tiles/mobil-app-tile.tsx

```diff
@@ -1,14 +1,31 @@
 import { MobilAppData } from '@/models/models';
 
+import { Tile } from './tile';
+
 type Props = {
   data: MobilAppData;
 };
 
-export function MobilAppTile({ data: { description } }: Props) {
+export function MobilAppTile({ data }: Props) {
   return (
-    <div className='tile sm:col-span-3 p-10'>
-      <h1>Konferenciapp</h1>
-      <p>{description}</p>
-    </div>
+    <Tile className='sm:col-span-6 md:col-span-3'>
+      <Tile.Body>
+        <h2 className='text-3xl sm:text-[34px]'>KonferenciApp</h2>
+        <div className='flex flex-col gap-4 lg:gap-0 lg:flex-row w-full justify-between mt-4 items-center'>
+          <img src='img/app-screens.png' className='object-cover rounded-3xl h-[300px] mr-3' alt='App screens' />
+          <p className='text-stone-200 text-[20px] whitespace-pre-line w-full h-full items-center'>
+            Töltsd le a mobil appot, hogy élőben köveshesd a konferencia eseményeit és kérdezhess az előadóktól!
```

Comment:
sorry, that's my fault, but there is a typo here: it should be 'követhesd', not 'köveshesd'

Reply:
Yea, that was not the intended look small screens.